assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -25,6 +25,7 @@ import reducer from './reducers/reducers';
 
 const participation = abTest.init();
 
+
```

Comment:
Excellent adherence to Jamie's linting rules here, I like it 👍 (I suspect @davidfurey isn't so keen).